### PR TITLE
Migrate CI/CD to GH Actions

### PR DESCRIPTION
**Pull request by Lusitaniae**

### CODE REVIEW

File: apache_exporter.go [hunk 1]
```diff
@@ -16,14 +16,14 @@ import (
 	"time"
 
 	"github.com/Lusitaniae/apache_exporter/collector"
+	kingpin "github.com/alecthomas/kingpin/v2"
```
Comment (blkperl) [inline apache_exporter.go:19]:
```suggestion
	kingpin "gopkg.in/alecthomas/kingpin.v2"
```
I think switching back to gopkg.in will solve the lint error.

**Comment by Lusitaniae** [comment]
great, thanks for the help @blkperl

**Comment by Lusitaniae** [comment]
I could consider adding you as a maintainer if you're interested/willing @blkperl ?

**Comment by Lusitaniae** [comment]
- Messy PR, should have made squash merge :disappointed: 
- Need to update the GH actions config to upload to my docker/quay accounts instead of the defaults prom

**Comment by blkperl** [comment]
@Lusitaniae , Yeah that sounds fine. I can contribute as a maintainer.

**Comment by blkperl** [comment]
@Lusitaniae sorry I went on vacation and the invite expired. Can you send it again?

**Comment by Lusitaniae** [comment]
Sent a new invite @blkperl

**Pull request merged**